tests/perf/pubsub_bulk_publish_http/pubsub_bulk_publish_http_test.go

```diff
@@ -0,0 +1,148 @@
+//go:build perf
+// +build perf
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pubsub_bulk_publish_http
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/dapr/dapr/tests/perf/utils"
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+	"github.com/dapr/dapr/tests/runner"
+	"github.com/dapr/dapr/tests/runner/loadtest"
+	"github.com/dapr/dapr/tests/runner/summary"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	tr *runner.TestRunner
+)
+
+const (
+	k6AppName = "k6-test-app"
```

Comment:
We had fortio bulk publish test till now. Do we want to remove that as a part of this PR or we want to let both k6 and fortio running for bulk publish?

Reply:
I will defer that to maintainers. Are we going to clean up fortio eventually and move to k6? We can also keep both tests, since Fortio measures gRPC and k6 measures HTTP.

Reply:
Makes sense.